.github/workflows/ci.yml

```diff
@@ -48,22 +48,19 @@ jobs:
         - use-conda: 'Yes'  # No conda packages yet for Qt6
           skip-pyqt6: true
           skip-pyside6: true
-        - python-version: '3.6'
+        - python-version: '3.7'
           use-conda: 'No'
-          skip-pip-check: 'true'  # Pytest packaging issue on Python 3.6 defaults channel
-          pyside2-version: '5.12.0'  # 5.12.1-5.12.6 fails on collection/segfaults on patch test
         - os: windows-latest
           python-version: '3.10'
           use-conda: 'No'
           pyside2-version: '5.15'  # No 5.12 wheel on Windows and Python 3.10
         - os: windows-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'Yes'
-          pyqt5-qt-version: '5.9'  # 5.12 is apparently unreliable here
+          pyqt5-qt-version: '5.9'
         - os: macos-latest
-          python-version: '3.6'
+          python-version: '3.7'
           use-conda: 'No'
```

Comment:
```suggestion
```

Likewise, this block doesn't do anything now as is and can simply be eliminated.